gateway/1.8/modules/ROOT/pages/_partials/task-config-readiness-liveness.adoc

```diff
@@ -0,0 +1,191 @@
+//tag::config-readiness-liveness-page-body[]
+
+Flex Gateway supports two methods for ensuring reliability and efficiency.
+
+* Readiness probes test whether a gateway instance is configured correctly and ready for incoming traffic. This ensures that end-users are not affected by unstable or partially started services. For information, refer to <<configure-a-readiness-probe>>.
+
+* Liveness probes test whether a gateway instance is operational. This minimizes downtime due to crashes or other failures. For information, refer to <<configure-a-liveness-probe>>.
+
+[[configure-a-readiness-probe]]
+== Configure a Readiness Probe
+
+To test whether a Flex Gateway instance is ready for incoming traffic, you can use the following CLI command:
+
+[source,ssh]
+----
+flexctl probe --check=readiness
+----
+
+If the Flex Gateway instance is ready to handle incoming traffic, the command returns an exit code of `0`. Otherwise the command returns an exit code of `1`.
+
+When Flex Gateway is started, its readiness state is `false`. The state changes to `true` after the following conditions are met:
+
+* API instances are deployed (at least one API must be configured)
+* Policies are initialized
+
+The readiness state is not impacted when you apply configuration changes to a running gateway instance. This ensures that the instance continues to receive traffic during configuration updates.
+
+Refer to the following for information about configuring readiness probes in specific environments:
+
+[tabs]
+====
+Kubernetes and OpenShift::
++
+By default, Flex Gateway running on Kubernetes includes a readiness probe configured in the Helm chart. The readiness probe automatically runs `flexctl probe --check=readiness --allow-api-errors --allow-envoy-errors --allow-policy-errors` at an interval of 10 seconds.
++
+The readiness probe is configured by default with the following values:
++
+[source,helm]
+----
+readinessProbe:
+ exec:
+   command:
+   - flexctl
+   - probe
+   - --check=readiness
+   - --allow-api-errors
+   - --allow-envoy-errors
+   - --allow-policy-errors
+ initialDelaySeconds: 10
+ periodSeconds: 10
+ failureThreshold: 2
+ timeoutSeconds: 5
+----
++
+[cols="1,2"]
+|===
+| Parameter | Description
+| `exec.command` | The readiness probe command
+| `--allow-api-errors`| The flag to allow errors in API instances
+| `--allow-envoy-errors`| The flag to allow Envoy configuration errors
+| `--allow-policy-errors`| The flag to allow policy configuration errors
+| `initialDelaySeconds` | The time in seconds to wait after startup before executing the first readiness probe
+| `periodSeconds` | The period in seconds between each readiness probe
+| `failureThreshold` | The number of failed readiness probes before the Kubernetes pod is restarted
```

Comment:
```suggestion
| `failureThreshold` | The number of failed readiness probes before the Kubernetes pod Ready condition is set to false.
```
The readiness check doesn't restart the pod. More info here: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes